firstvoices/backend/tests/test_tasks/test_build_mtd.py

```diff
@@ -211,13 +212,49 @@ def test_build_and_score_bad_image_data(self, site, caplog):
     def test_old_results_removed(self, site):
         build_index_and_calculate_scores(site.slug)
         build_index_and_calculate_scores(site.slug)
-        final_result = build_index_and_calculate_scores(site.slug)
+        final_result = build_index_and_calculate_scores(site.slug).export_result
 
         # Check that only the most recent is in the db
         saved_results = MTDExportFormat.objects.filter(site=site)
         assert len(saved_results) == 1
-        assert saved_results.latest().latest_export_result == final_result
-        assert not saved_results.latest().is_preview
+        assert saved_results.latest().export_result == final_result
+
+    @pytest.mark.django_db
+    def test_build_and_score_not_triggered_while_running(self, site, caplog):
```

Comment:
One more optional name change suggestion, `test_parallel_build_and_score_jobs_not_allowed` for this test